.github/workflows/multi-arch-gating.yaml

```diff
@@ -0,0 +1,66 @@
+name: Mult-Arch-Gating
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build-image:
+    name: Build Cachi2 image and run smoketest on it
```

Comment:
If we drop the tests the names should be updated accordingly

Reply:
Have renamed the workflow with multi-arch-build